config/routes.rb

```diff
@@ -13,6 +13,7 @@
   resources :purchases, only:[:create]
   resources :carts, only:[:show]
 
+  post '/orders/:id', to: 'orders#create'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.